6213 МакаровСА Лаб. 4

123321zZzZ opened the pull request.

123321zZzZ committed:
lab1_var12

Lab 1 completed

123321zZzZ committed:
corrections

corrections

123321zZzZ committed:
lab_2_v12

Lab 2 completed

123321zZzZ committed:
correction

fix: исправить именование классов по БЭМ
refactor: использовать вложенные селекторы для стилизации ссылок

123321zZzZ committed:
fix:

fix:  new name

pl_pl_list_item - > pl-list__item
pl_pl_list_link -> item__link
parent-3-black -> parent-3

123321zZzZ committed:
feat: add resume HTML structure

123321zZzZ committed:
feat: add and configure CSS styling

123321zZzZ committed:
feat: add photos to gallery and assets folder

123321zZzZ committed:
Добавлена папка .idea (конфигурация WebStorm)

123321zZzZ committed:
Добавлен Html Лабораторной работы №4

123321zZzZ committed:
Добавлена логика калькулятора

123321zZzZ committed:
Добавлены стили

123321zZzZ committed:
Добавление кнопки очистки полей ввода

123321zZzZ committed:
Добавление логики кнопки очистки полей ввода , сделан вывод ошибок через браузер

123321zZzZ committed:
Создание раздела (Мои проекты)

123321zZzZ committed:
Добавление стилей для раздела (мои проекты)

AvtoBBus reviewed (changes requested):
Все `getElementById` могут не найти элемент поэтому нужно проверять, что элемент найден и только потом работать с ним

123321zZzZ committed:
add БЕМ

123321zZzZ committed:
rename

123321zZzZ committed:
fix

Добавлены проверки наличия элементов через getElementById, чтобы предотвратить ошибки, если какой-либо элемент отсутствует на странице.

Также убрана глобальная переменная currentIndex — выбранная операция теперь хранится в атрибуте элемента selectedOp.

AvtoBBus commented:
Принято
